resources/markdown/Sync Salesforce Leads between accounts and check and record state_instructions.md

```diff
@@ -0,0 +1,7 @@
+1. Clicking **Create flow** takes you to the App Connect flow editor.
```

Comment:
We've just updated the format of the instructions, so please replace this with:

To refer to these instructions while editing the flow, open [the github page](https://github.com/ot4i/app-connect-templates/blob/master/resources/markdown/Sync%20Salesforce%20Leads%20between%20accounts%20and%20check%20and%20record%20state_instructions.md) (opens in a new window).

1. Click **Create flow** to start using the template.
1. Connect to your source [Salesforce account](https://developer.ibm.com/integration/docs/app-connect/how-to-guides-for-apps/use-ibm-app-connect-salesforce/).
1. Connect to your target Salesforce account.
1. Connect to your Gmail account and update the email subject and body to meet your business needs. 
1. Connect to your [Slack account](https://developer.ibm.com/integration/docs/app-connect/how-to-guides-for-apps/use-ibm-app-connect-slack/) and select a channel where you want batch level error messages to be sent.
1. To start the flow, in the banner open the options menu [&#8942;] then click **Start flow**.